client/app/bundles/course/assessment/submission/components/ScribingView/ToolDropdown.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import FontIcon from 'material-ui/FontIcon';
+import MaterialTooltip from 'material-ui/internal/Tooltip';
+
+const propTypes = {
+  toolType: PropTypes.string.isRequired,
+  tooltip: PropTypes.string,
+  showTooltip: PropTypes.bool,
+  currentTool: PropTypes.string.isRequired,
+  onClick: PropTypes.func,
+  onClickIcon: PropTypes.func,
+  onTouchTapChevron: PropTypes.func,
+  colorBar: PropTypes.string,
+  iconClassname: PropTypes.string,
+  colorBarComponent: PropTypes.func,
+  iconComponent: PropTypes.func,
+  onMouseEnter: PropTypes.func,
+  onMouseLeave: PropTypes.func,
+  popoverComponent: PropTypes.func,
+};
+
+const style = {
+  tool: {
+    position: 'relative',
+    display: 'inline-block',
+    paddingRight: '24px',
+    outline: 'none',
+  },
+  innerTool: {
+    display: 'inline-block',
+    outline: 'none',
+  },
+  colorBar: {
+    width: '23px',
+    height: '8px',
+  },
+  chevron: {
+    color: 'rgba(0, 0, 0, 0.4)',
+    fontSize: '12px',
+    padding: '10px 0px 10px 0px',
+  },
+};
+
+export default class ToolDropdown extends Component {
+  renderIcon() {
+    const { iconClassname, currentTool, toolType, iconComponent } = this.props;
+
+    return iconComponent ?
+      iconComponent() :
+      <FontIcon
+        className={iconClassname}
+        style={
+          currentTool === toolType ?
+            { color: 'black' } :
```

Comment:
> Would blue or some other colour for the selected component be better?
Sounds good to me. I was using black as the selected color because black was the default hover color for Material UI icon buttons but we can change that.